helmfile/charts/gatekeeper/podsecuritypolicies/templates/_.tpl

```diff
@@ -0,0 +1,39 @@
+{{/* Render exceptions */}}
```

Comment:
Any reason for why just calling this `_`?

Reply:
It just the convention I've rolled with.

Reply:
Ok sure